test/include/test_util_conversion.h

```diff
@@ -626,17 +626,114 @@ static std::string asText(const ble_gap_adv_report_type_t &reportType)
 
     return retval.str();
 }
-#endif
 
-#if NRF_SD_BLE_API == 6
 static std::string asText(const ble_gap_aux_pointer_t &auxPointer)
 {
     std::stringstream retval;
     retval << "aux_offset:" << static_cast<uint32_t>(auxPointer.aux_offset);
     retval << " aux_phy:" << static_cast<uint32_t>(auxPointer.aux_phy);
     return retval.str();
 }
-#endif
+
+static std::string asText(const ble_gap_phys_t &phys)
```

Comment:
Are these event-to-text conversions intended to be used only in test context? Could be useful to include in log file also for normal operation.

Reply:
@bihanssen, currently they are only in the tests. Adding them in pc-ble-driver log framework is something we could do. Perhaps a better solution is to add them in a separate C++ library that encapsulates the pc-ble-driver.

Reply:
Ok, something for the backlog then, Good logging is certainly valuable to have.